SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -310,8 +310,8 @@ def customise_Reco(process,pileup):
     # add the correct tracking back in
     process.load("RecoTracker.Configuration.RecoTrackerPhase1PU"+str(nPU)+"_cff")
 
-    process.globalreco.insert(itIndex,process.trackingGlobalReco)
-    process.reconstruction.insert(grIndex,process.globalreco)
+    process.globalreco_tracking.insert(itIndex,process.trackingGlobalReco)

```

Comment:
@mark-grimes I guess these changes should not interfere with the phase1 era migration, but FYI nevertheless.